src/test/java/javatests/JsonConverterTest.java

```diff
@@ -94,4 +94,30 @@ public void testUpdateFileMethod() throws IOException {
     assertTrue("updateFile() didn't return true when it should have.", 
       expectedResponse == actualResponse);
   }
+
+  /** Test that getConfig() returns the expected file content. */
+  @Test
+  public void testGetConfigMethod() {
+    JsonConverter converter = new JsonConverter();
+    String vendorID = "vend_1";
+    String expectedResponse = "{\"Vehicle\":{\"Car\":\"Blue Tacoma\"}}";
+
+    String actualResponse = converter.getConfig(vendorID);
+
+    assertTrue("getConfig() didn't return the expected file content.",
+      expectedResponse.equals(actualResponse));
+  }
```

Comment:
Would it make sense to show the difference in output here?

Reply:
I think print statements would be helpful if I was failing this test. Since this test works as expected, showing the difference doesn't make sense because there is no difference in the results.

Reply:
sgtm

Reply:
Consider adding a comment to this test explaining that there must be a file on the filesystem with this name and contents. 

Otherwise, consider actually writing the file in this test and then deleting it when you're done.